src/actions/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Action types
+ */
+
+export const LOGIN_USER = 'LOGIN_USER'
```

Comment:
Consider changing this variable name. It's kinda hard to understand what it's meant for. How about `LOGIN_STATUS`.